### PR TITLE
Fix cannot update changed messages in the dictionary

### PR DESCRIPTION
**Pull request by bbuhler**
Hello (:

This is a fix for #107. It clears the `lookupCache` for a `locale` when `addMessages` is called.

### Before submitting the PR, please make sure you do the following

- [x] It's really useful if your PR relates to an outstanding issue, so please reference it in your PR, or create an explanatory one for discussion. In many cases features are absent for a reason.
- [x] This message body should clearly illustrate what problems it solves. If there are related issues, remember to reference them.
- [x] Ideally, include a test that fails without this PR but passes with it. PRs will only be merged once they pass CI. (Remember to `npm run lint`!)

### Tests

- [x] Run the tests tests with `npm test` or `yarn test`

### CODE REVIEW

**Comment by bbuhler** [comment]
I just found out that you have already a related branch (https://github.com/kaisermann/svelte-i18n/tree/fix/lookup-caching).

Though my solution is a quick'n'dirty, it would unblock me with my work.

**Comment by bbuhler** [comment]
@kaisermann can you please merge this or give any feedback (:

File: test/runtime/includes/lookup.test.ts [hunk 1]
```diff
@@ -83,3 +83,11 @@ test("doesn't cache falsy messages", () => {
...
   });
 });
+
+test('updates message of a locale dictionary', () => {
+  addMessages('en', { field: 'name' });
+  expect(lookup('field', 'en')).toBe('name');
+
+  addMessages('en', { field: 'name2' });
+  expect(lookup('field', 'en')).toBe('name2');
+});
```
Comment (kaisermann) [inline test/runtime/includes/lookup.test.ts:93]: This test name doesn't seem to be related to what we're actually testing. Maybe:

```suggestion
test('clears a locale lookup cache when new messages are added', () => {
  addMessages('en', { field: 'name' });
  expect(lookup('field', 'en')).toBe('name');

  addMessages('en', { field: 'name2' });
  expect(lookup('field', 'en')).toBe('name2');
});
```

**Review by kaisermann — APPROVED** [review]

**Comment by kaisermann** [comment]
Hey @bbuhler 👋  I'm not sure if clearing the whole locale cache is the best way to do this, but I don't have an alternative right now. thanks!
